.github/workflows/docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Docs Generation
+
+on:
```

Comment:
After https://github.com/ably/sdk-upload-action/pull/40 is merged and released, we'll want to make this workflow run for all tags, too, so that we're permanently saving documentation for each release.

```yaml
tags:
    "*"
```

But you can do it as a separate PR later.

Reply:
I've made an issue for that https://github.com/ably/ably-cocoa/issues/1440